extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -205,7 +208,10 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
             assetIcons={assetIcons}
             sortedBalances={[
               {
-                token: { issuer: sourceAsset.issuer, code: sourceAsset.code },
+                token: {
```

Comment:
related fix :), AccountBalances expects the issuer to be nested one more level under the `key` field